iron-dropdown.html

```diff
@@ -212,6 +212,13 @@
         },
 
         /**
+         * Whether the text direction is RTL
+         */
+        _isRTL: function() {
```

Comment:
This implementation is gonna get repeated in so many places, I wish we had an `IronI18nBehavior` or similar..

No action required, just a gripe.